docs/governance-and-management/governance/governance-processes.md

```diff
@@ -66,3 +66,30 @@ Types of technical design decisions that require approval:
     |**Product Design**|Consulted|Consulted|Consulted|Consulted|
     |**Technical Design**|Responsible|Responsible|Responsible|Accountable|
 
+### Technical Design Process
+
+This section outlines the process for features that require a technical design document. The purpose of a design document is to draft and approve major technical design decisions to ensure the quality, reliability, and maintainability of CiviForm for all its users.
+
+#### Before You Start
+
+The following should be true:
+1. There is an open issue in [GitHub](https://github.com/civiform/civiform/issues)
+2. There is an approved PRD in the [public folder)(https://drive.google.com/drive/u/0/folders/1gqw3VSK-lMPgRfcS_ZEhF3C_51We0eGz)
+3. If the issue has a `UX` label, approved UX Mocks are added to the [public folder](https://drive.google.com/drive/u/0/folders/1zY6OnQYH7Xo5ebPctTJtd5QRsHBAI5vF)
+4. A `Ready for TDD` label is added to the issue and visible in [this list](https://github.com/civiform/civiform/issues?q=is%3Aissue+is%3Aopen+label%3AReady+for+TDD)
+
+During the `CiviForm eng weekly` meeting on Mondays. We review the issues with the `Ready for TDD` label and assign an author, approvers and reviewers. Assignment can be during the week either in a daily status meeting or by discussing in the [engineering slack channel](https://civiform.slack.com/archives/C01QKN2UNMA).
```

Comment:
```suggestion
During the `CiviForm eng weekly` meeting on Mondays we review the issues with the `Ready for TDD` label and assign an author, approvers and reviewers. Assignment can be during the week either in a daily status meeting or by discussing in the [engineering slack channel](https://civiform.slack.com/archives/C01QKN2UNMA).
```

Reply:
nit: usually like to reserve the backticks for code-related things vs names like the name of the meeting

Reply:
issue (blocking): it's not clear from these sentences the difference between things assigned at the weekly eng meeting vs the daily status meeting and slack discussions

Reply:
There isn't any difference, other than I'm suggesting that there will be dedicated time in the weekly eng meeting and others can be ad hoc, if we don't want to wait for the next meeting. I'm also fine with saying that it only happens on the Monday meetings, but that feels like it's unnecessarily blocking. I can make that more clear though.

Reply:
Updated the language. I imagine that this section will be iterated on as it gets put into practice.